src/Pages/Explore/JobSlide/SlideFliter/CareerModal.js

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+import styled from "styled-components";
+import SlideOff from "./SlideSvg";
+const CareerModal = (props) => {
```

Comment:
```suggestion
import SlideOff from "./SlideSvg";

const CareerModal = ({ display, careerHandler }) => {
```
- import랑 컴포넌트 사이 한줄 띄워주세요
- 함수 컴포넌트 파라미터 자리에 직접 구조분해 할당 가능합니다